app/src/main/res/values/speech_translation.xml

```diff
@@ -11,5 +11,5 @@
     <string
         name="speechToSpeechEndpoint"
         formatted="false"
-        translatable="false">https://us-central1-YOUR-PROJECT-ID.cloudfunctions.net/speechTranslate</string>
+        translatable="false">https://us-central1-qwiklab-gke-debugging.cloudfunctions.net/speechTranslate</string>
```

Comment:
We should not hard code the project ID here. This value should come from the speech_translation_test.xml file that is copied as part of the testing environment. This is related to my previous comment. What was the problem with the `speech_translation_test.xml` file?

Reply:
Updated the file in our testing environment.